src/shared-api-objects/user-has-access-react.component.tsx

```diff
@@ -16,7 +16,11 @@ export default function UserHasAccessReact(props) {
   }, []);
 
   if (user) {
-    return userHasAccess(props.privilege, user) !== undefined && props.children;
+    return (
+      props.privileges.some(
```

Comment:
Renaming the prop from `privilege` to `privileges` is a breaking change. We'd need to bump the version up to 2.0.0 and also update the places that use the `UserHasAccessReact` component so that everything still works.

Alternatively, we could still use the `privilege` prop, but detect if it's a string or array.

I am okay with either approach - it's not a bad thing to bump up the major version, but it's something worth knowing before merging this in. What are your thoughts on how to handle it?